build.gradle

```diff
@@ -68,19 +72,42 @@ dependencies {
     deobfCompile "net.sengir.forestry:forestry_1.12.2:5.8.0.277+"
     deobfCompile "net.industrial-craft:industrialcraft-2:2.8.75-ex112"
     deobfCompile "slimeknights.mantle:Mantle:1.12-1.3.2.26"
+    runtime "reborncore:RebornCore-1.12.2:3.16.4.478:universal"
     deobfCompile "steves-carts-reborn:StevesCarts:1.12.2:2.4.18.92"
     deobfCompile "li.cil.oc:OpenComputers:MC1.12.1-1.7.+"
     deobfCompile "moo-fluids:MooFluids:1.12.2:1.7.11.02a"
     deobfCompile "com.wayoftime.bloodmagic:BloodMagic:1.12.2-2.2.+"
     deobfCompile "roost:roost:1.12:1.3.0"
     deobfCompile "mystical-agriculture:MysticalAgriculture:1.12.2:1.7.3"
+    deobfCompile "lycanites-mobs:lycanitesmobs:1.12.2:2.0.1.2"
+    deobfCompile "rustic:rustic:1.1.1"
+    runtime "team.chisel.ctm:CTM:MC1.12.2-0.3.3.22"
+    deobfCompile "chisel:Chisel:MC1.12.2:1.0.1.44"
+    deobfCompile "storage-drawers:StorageDrawers:1.12.2:5.4.0"
+    deobfCompile "ice-and-fire-dragons:iceandfire:1.8.3"
+    deobfCompile "architecturecraft-tridev:architecturecraft:1.12:3.98"
+    deobfCompile "unlimited-chisel-works:UnlimitedChiselWorks:0.3.0"
+    compile "advanced-generators:generators-0.9.20.12:mc1.12.2:dev"
+    runtime "codechicken:CodeChickenLib:1.12.2-3.2+:universal"
+    runtime "cofh:CoFHWorld:1.12.2-1.2+:universal"
+    runtime "cofh:ThermalFoundation:1.12.2-2.6.3.27:universal"
+    compile "cofh:ThermalExpansion:1.12.2-5.5.4.43:deobf"
+    compile "cofh:ThermalDynamics:1.12.2-2.5.5.21:deobf"
+    compile "bdlib:bdlib-1.14.3.12:mc1.12.2:dev"
     runtime "cucumber:Cucumber:1.12.2:1.1.3"
     runtime "info.amerifrance.guideapi:Guide-API:1.12-2.1.+"
-    compile "cofh:RedstoneFlux:1.12-2.0.1.2:universal"
+    compile "chameleon:Chameleon:1.12:4.1.3"
+    compile "llibrary:llibrary:1.7.9:1.12.2"
+    compile "cofh:CoFHCore:1.12.2-4.6.3.27:deobf"
+    compile "cofh:RedstoneFlux:1.12-2.1.0.+:universal"
     //Storage drawers, Botania, Chisel, CoFH+, Advanced Generators, Lycanite's Mobs
     compileOnly fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Maybe group some of these together so it's more clear what the runtime dependencies are for.